board_client/animations/trench_run.py

```
@@ -19,303 +21,337 @@
+
             for laser in lasers:
-                laser["progress"] += 0.1
+                laser["progress"] += 0.12
                 prog = min(1.0, laser["progress"])
-                start_x, start_y = project(laser["start_x"], 0.3, laser["z"])
-                current_x = laser["start_x"] + (laser["end_x"] - laser["start_x"]) * prog
-                current_point_x, current_point_y = project(current_x, 0.3, laser["z"])
-                graphics.set_pen(graphics.create_pen(255, 30, 30))
-                graphics.line(start_x, start_y, current_point_x, current_point_y)
+                tower = laser["source_tower"]
+                start_x, start_y = project_flight(
```

Pylint (3.3.7)

[error] Parsing failed: ''(' was never closed (trench_run, line 280)'

(E0001)